exercises/APE5-6/APE05-06E1d.tex

```diff
@@ -0,0 +1,125 @@
+Unfortunately we don't have strongly dominated strategies in this game so we have a lot of support to check. Let's begin with pure strategies. We can find 2 Nash equilibria :
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{z_2\}}$. Indeed step 3 is verified :
+\begin{align*}
+    8 =w_1&\ge u_1(y_1,z_2)=5   \\
+    7=w_2&\ge u_2(x_1,y_2)=5 \quad \mbox{and} \quad 7=w_2\ge u_2(x_1,x_2)=4 
+\end{align*}
+
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{x_2\}}$. Indeed step 3 is verified :
+\begin{align*}
+    2 =w_1&\ge u_1(x_1,x_2)=0 \\
+    9=w_2&\ge u_2(y_1,y_2)=5  \quad \mbox{and} \quad  9=w_2\ge u_2(y_1,z_2)=1
+\end{align*}
+
+\begin{itemize}
+  \item[$\bullet$] Randomized vs pure strategy supports
+\end{itemize}
+
+Fortunately we can quickly check these support with the payoff equations (step 2). We can also notice that for example if the support $\mathbf{\{x_1\}}\times\mathbf{\{x_2,y_2\}}$  does not lead to a Nash equilibrium, the support $\mathbf{\{x_1\}}\times\mathbf{\{x_2,y_2,z_2\}}$ does not lead to a Nash equilibrium too. 
+
+- $\mathbf{\{x_1,y1\}}\times\mathbf{\{x_2\}}$, $\mathbf{\{x_1,y1\}}\times\mathbf{\{y_2\}}$ and $\mathbf{\{x_1,y1\}}\times\mathbf{\{z_2\}}$ does not lead to a Nash equilibria, indeed :
+\begin{equation*}
+    0\ne2 \quad 5\ne6 \quad 8\ne5
+\end{equation*}
+
+- $\mathbf{\{x_1\}}\times\mathbf{\{x_2,y_2\}}$, $\mathbf{\{x_1\}}\times\mathbf{\{x_2,z_2\}}$ and $\mathbf{\{x_1\}}\times\mathbf{\{y_2,z_2\}}$ does not lead to a Nash equilibria, indeed :
+\begin{equation*}
+    4\ne6 \quad 4\ne7 \quad 6\ne7
+\end{equation*}
+
+- $\mathbf{\{y_1\}}\times\mathbf{\{x_2,y_2\}}$, $\mathbf{\{y_1\}}\times\mathbf{\{x_2,z_2\}}$ and $\mathbf{\{y_1\}}\times\mathbf{\{y_2,z_2\}}$ does not lead to a Nash equilibria, indeed :
+\begin{equation*}
+    9\ne5 \quad 9\ne1 \quad 5\ne1
+\end{equation*}
+
+
+And so as explain $\mathbf{\{x_1\}}\times\mathbf{\{x_2,y_2,z_2\}}$ and $\mathbf{\{y_1\}}\times\mathbf{\{x_2,y_2,z_2\}}$ does not lead to a Nash equilibria.
```

Comment:
"to a Nash equilibria" -> "to Nash equilibria". By the way, it is either "several Nash equilibria" or "one Nash equilibrium". you can find and replace in your solutions to fix all the "a Nash equilibria" to "a Nash equilibrium"